swift-generator/src/main/java/com/facebook/swift/generator/ConstantRenderer.java

```diff
@@ -200,7 +201,7 @@ private String renderIntegerConstant(BaseType baseType, ConstInteger value)
 
             case I64:
                 return Long.toString(value.value()) + "L";
-
+                
```

Comment:
I think what he means is an empty newline is fine, but this one has a tab in it.